proposals/0207-raise-block-limits-to-50M.md

```diff
@@ -0,0 +1,92 @@
+---
+simd: '0207'
+title: Raise Block Limits to 50M CUs
+authors:
+  - Andrew Fitzgerad (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2024-12-05
+feature:
+supersedes:
+superseded-by:
+extends:
+---
+
+## Summary
+
+Raise the block limit from 48M to 50M CUs.
+
+## Motivation
+
+Current block limits are set to 48M CUs.
+Block limits's primary purpose is to make sure that the vast majority of
+network participants are able to keep up with the network, by restricting the
+amount of work a leader is allowed to pack into a block.
+However, current main net beta traffic is largely not constrained by large
+block execution times.
+This proposal aims a modest increase in block limits to 50M CUs, in order to
+give some additional capacity to the network, and client implementations ready
+for future increases as the performance of the network improves.
+
+## New Terminology
+
+No new terms, but the following definitions are given for clarity:
+
+- Max Block Units - The maximum number of CUs that can be included in a block.
+- Max Writable Account Units - The maximum number of CUs that can be used to
+  write any given account in a block.
+- Max Vote Units - The maximum number of CUs that can be used by simple votes
+  in a block.
+- Max Block Accounts Data Size Delta - The maximum size of the accounts data
+  size delta in a block.
+
+## Detailed Design
+
+The following table shows the current block limits and the proposed block
+limits:
+
+| Type | Current Block Limit | Proposed Block Limit |
+|------|----------------------|----------------------|
+| Max Block Units | 48M | 50M |
+| Max Writable Account Units | 12M | 12M |
+| Max Vote Units | 36M | 36M |
+| Max Block Accounts Data Size Delta | 100MB | 100MB |
```

Comment:
What does increasing Block Accounts Data Size Delta affect ?

Reply:
We're not increasing it. It limits overall state growth.